shopinvader_cart_reminder/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Shopinvader cart reminder",
+    "description": """Shopinvader module to relaunch the customer when the
```

Comment:
Can you use the new READMe structure (based on a readme folder)

Moreover The description should be a little more complete. We should explain that once the module is installed an email is sent to a customer if a pending cart without activity into a specific period is found. We should also mention that the period and the mail template are configurable.